addons/mail/static/src/core/web/chatter.js

```diff
@@ -301,7 +293,7 @@ export class Chatter extends Component {
     onFollowerChanged() {
         document.body.click(); // hack to close dropdown
         this.reloadParentView();
-        this.load(this.props.threadId, ["followers", "suggestedRecipients"]);
+        this.load(this.state.thread, ["followers", "suggestedRecipients"]);
     }
 
     onPostCallback() {
```

Comment:
same for all others, whenever there is async/callback, the original thread should be passed on